app/src/main/java/com/framgia/fsalon/screen/report/ReportViewModel.java

```diff
@@ -88,4 +78,15 @@ public void onStop() {
     public void setPresenter(ReportContract.Presenter presenter) {
         mPresenter = presenter;
     }
+
+    /**
+     * TabButton
+     */
+    @IntDef({TAB_SALES, TAB_CUSTOMERS, TAB_BILLS})
+    @Retention(RetentionPolicy.SOURCE)
+    public @interface TabButton {
+        int TAB_SALES = 0;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.